webpack.config.js

```diff
@@ -59,6 +59,29 @@ module.exports = (shell = {}) => {
             },
           ],
         },
+        {
+          test: /\.(js|jsx|mjs)$/,
```

Comment:
Interestingly, this doesn't appear to do anything, maybe because our prop-type versions matches, so they are removed in 1 go.

Any idea if this is working, since it has no affect?

Reply:
Investigating this and seems you're right @dr3  - the existing webpack rule having `include` set to `/src/` seems highly suspicious

I assume the intention of that pattern was to restrict webpack to the `src/` directory of simorgh, but I believe it might be including every path that includes the pattern `src` in it, which includes psammead node modules (and presumably other node modules too)

Will investigate 🤔

Reply:
Yep 🤦‍♂ 
- put some JSX in a file that doesn't have src and import it - webpack crashes
- rename the file to `src` - webpack compiles

This should now be fixed so only things in `/src` get transpiled by the `.babelrc` rules - things in `node-modules/@bbc` get transpiled by these rules in the webpack config